lib/streamlit/commands/execution_control.py

```diff
@@ -80,7 +80,7 @@ def rerun() -> NoReturn:  # type: ignore[misc]
 def experimental_rerun() -> NoReturn:
     """Rerun the script immediately.
 
-    When `st.experimental_rerun()` is called, the script is halted - no
+    When ``st.experimental_rerun()`` is called, the script is halted - no
```

Comment:
nit: we might also need the double quotes for the rerun method above?

Reply:
Added.